gcloud/search/test_index.py

```diff
@@ -105,9 +105,168 @@ def test_from_api_repr_w_properties(self):
         self.assertTrue(index._client is client)
         self._verifyResourceProperties(index, RESOURCE)
 
+    def test_list_documents_defaults(self):
```

Comment:
These tests always feel so long. Do you think there's anything we can do to make them shorter?